packages/patternfly-react/src/components/Masthead/Masthead.js

```diff
@@ -16,11 +16,20 @@ const Masthead = ({
   navToggle,
   onNavToggleClick,
   middleContent,
+  fit,
   children,
   ...props
 }) => {
-  const mastheadClasses = classNames('navbar', 'navbar-pf-vertical', className);
-
+  let mastheadClasses = classNames('navbar', className);
+  if (fit) {
+    mastheadClasses = classNames(
+      'navbar-default',
+      'navbar-pf',
+      mastheadClasses
+    );
+  } else {
+    mastheadClasses = classNames('navbar-pf-vertical', mastheadClasses);
+  }
```

Comment:
This can be simplified to:
```
  const mastheadClasses = classNames(
    {
      navbar: true,
      'navbar-default': fit,
      'navbar-pf': fit,
      'navbar-pf-vertical': !fit
    },
    className
  );
```